.travis.yml

```diff
@@ -19,6 +19,7 @@ env:
     - CCACHE_COMPRESS=1
     - CC="clang"
     - CXX="clang++"
+    - HOST_OS="$(if [ "$TRAVIS_OS_NAME" == "osx" ]; then echo "darwin"; else echo "$TRAVIS_OS_NAME")"
```

Comment:
Do you know of an alternative? To set `HOST_OS` to `darwin` of `TRAVIS_OS_NAME` is `osx`?